packages/react/src/GadgetProvider.tsx

```diff
@@ -48,15 +58,20 @@ export interface DeprecatedProviderProps {
  *
  * This component is __required__ as a wrapper around any React components using the other hooks in this library, like `useFindMany` or `useAction`.
  *
- * Pass an instance of your app's api client to the `value` prop.
+ * Pass an instance of your app's api client to the `api` prop.
  *
  * @example
- * <Provider value={api}>
+ * <Provider api={api}>
  *   <MyApp />
  * </Provider>
  *
  * @example the Provider accepts the deprecated form of passing an urql client object right in -- this is deprecated and will be removed in a future version. Instead, just pass the whole api instance.
- * <Provider value={api.connection.currentClient}>
+ * <Provider api={api.connection.currentClient}>
+ *   <MyApp />
+ * </Provider>
+ *
+ * @example the Provider accepts option sign in and sign out paths.
+ * <Provider api={api.connection.currentClient} signInPath="/auth/signin" signOutPath="/auth/signout">
```

Comment:
Do you think we're gonna add more paths over time, such that we should make this API `paths={{signIn: ..., signOut: ...}}`?

Reply:
I guess at that point it's borderline a `configuration` block. To keep it most flexible long term we could consider making the `auth` config it's own property: 

```tsx
<Provider api={api} auth={{ signInPath, signOutPath}} />
```